common/fileview2/fileview.hpp

```diff
@@ -139,10 +139,12 @@ interface IResultSetFactory : extends IInterface
 {
     virtual INewResultSet * createNewResultSet(IConstWUResult * wuResult, const char * wuid) = 0;
     virtual INewResultSet * createNewFileResultSet(const char * logicalFile, const char * cluster) = 0;
+    virtual INewResultSet * createNewFileResultSet(IDistributedFile * df, const char * cluster) = 0;
     virtual INewResultSet * createNewResultSet(const char * wuid, unsigned sequence, const char * name) = 0;
     virtual INewResultSet * createNewFileResultSet(const char * logicalFile) = 0;
     virtual IResultSetMetaData * createResultSetMeta(IConstWUResult * wuResult) = 0;
     virtual IResultSetMetaData * createResultSetMeta(const char * wuid, unsigned sequence, const char * name) = 0;
+    virtual IDistributedFile * lookupLogicalName(const char * logicalName) = 0;
```

Comment:
this is/was an internal helper, I don't think it should be exposed in the interface.